extensions/amazon-lambda/runtime/pom.xml

```diff
@@ -35,6 +35,17 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>3.0.0</version>
+            <scope>test</scope>
```

Comment:
I think we already have other mocking library declared or maybe this one. Could you check the bom/build-parent and see if we can avoid having 3 different mocking library?

Reply:
I could only find mockito as dependency in devtools. Didn't find any other mocking frameworks to use inside unit tests, except for mocking http / ... but that's a different purpose.

Reply:
OK, looks like I misremember. I would have sworn having seen at least Powermock. Sorry about that!

The version should go in the bom though.